### PR TITLE
Add the calculator design

### PR DESCRIPTION
**Pull request by Trust-007**
# Add the calculator design to project
- Deleted unused files and designs
- Created components folder
- Created the Calculator.js file and imported it in App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (TadesseDev) [inline src/App.js]: - [ ] Impressive job @Trust-007, One thing you forget here is to make the `App` component is a `class-based` component. This is to make sure you understand building a react app with class-based components.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Trust-007 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

You are almost don, Almost!!
There are some issues that you still need to work on to go to the next project but you are almost there!
EquatorialVictoriousBlackpanther-size_restricted

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
- [ ] Dear @Trust-007, on your public directory, there is still some boilerplate that got created by the CRA, please make sure to delete all including the `favicon.ico` and other unused files. This will help you create a clean project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Trust-007) [reply src/App.js]: Thanks for the suggestions @Tadesse-Alemayehu

**Review by muneebulrehman — APPROVED** [review]
Hi @Trust-007 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Working project

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**